Kuzzle/Utils/QueryUtils.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace KuzzleSdk.Utils {
+  public class QueryUtils {
+
+    public static void HandleRefreshOption(JObject query, bool waitForRefresh) {
+      if (waitForRefresh) {
+        query.Add("refresh", "wait_for");
+      }
+    }
+
+    public static void HandleNotifyOption(JObject query, bool notify) {
+      if (notify) {
+        query.Add("notify", true);
+      }
```

Comment:
If we happen to keep this util class, couldn't this method call just be replaced by `query.Add("notify", notify)`? 
I don't see any situation where setting it to false in the `JObject` could cause any harm.

Reply:
This is true, so I've changed it to do `query.Add("notify", notify)` instead.